src/main/java/sirius/biz/translations/BasicTranslations.java

```diff
@@ -111,15 +112,23 @@ public void updateText(@Nonnull Mapping field, String lang, String text) {
         }
 
         T translation = findOrCreateTranslation(field, lang, text);
-        translation.getTranslationData().setText(text);
-        updateTranslation(translation);
-
-        if (owner instanceof Journaled) {
+        if (translation != null) {
+            translation.getTranslationData().setText(text);
+            updateTranslation(translation);
+            if (owner instanceof Journaled) {
+                JournalData.addJournalEntry(owner,
+                                            String.format("Updated translated text for %s (%s): '%s'",
+                                                          field.getName(),
+                                                          lang,
+                                                          text));
+            }
+        } else if (owner instanceof Journaled) {
             JournalData.addJournalEntry(owner,
```

Comment:
Humm, JournalData is not really the place to log errors.

Reply:
the isValidLanguage() check in findOrCreateTranslation already produces a Handled exception (and returns null) if the language is invalid. So yeah, maybe the journal entry isn't needed then?